### ISSUE TITLE
loadings plot: visualization for factor space (loadings) (2D * x)

### ISSUE THREAD

**Issue by maxheld83** [aiorazabala/qmethod#94]
I'd also like to get a visualization of the factor space, with people (q sorts) located according to their loadings.
Obviously, this becomes quite complicated with `nfactors > 2`, and bears on the issue of rotation.

**Comment by maxheld83** [comment]
I'm now splitting this into two possible plots

- one **static** and **2D** (times x, depending on the number of factors) -> #94
- one **interactive** and **3D** -> #161

Some thoughts on this:

I *originally* thought interactive and 3D would be the way to go, and offer a great interface for manual rotation #13 – but now I'm not so sure.

Cons:
- 3D doesn't scale: you can't do that for *more* than 3 factors
- even in 3D, the display is unwieldy and overwhelming
- 3D is hard to publish
- packages rely on base plots, hard to add Q conveniences (labels, colors, etc). (I like my ggplot2).

So bottom line: I'm only going to do 2D plots for now.

Some examples for 3d plots in #161

**Comment by maxheld83** [comment]
like so:

![loaplot-ex](https://cloud.githubusercontent.com/assets/5372770/8511947/4d407cd2-232d-11e5-891e-297ae98d12f7.png)

and so, with a grid of all *combinations* (not permutations) of factors:

![loaplot-complete](https://cloud.githubusercontent.com/assets/5372770/8511956/97f81f28-232d-11e5-9ba0-0dd922ed5070.png)
